app/mm-redux/utils/channel_utils.ts

```diff
@@ -713,7 +713,7 @@ function buildUnreadChannels(channels: Array<Channel>, members: RelationOneToOne
         sort(sortChannelsByDisplayName.bind(null, locale));
 }
 
-function getUserLocale(userId: string, profiles: IDMappedObjects<UserProfile>) {
+export function getUserLocale(userId: string, profiles: IDMappedObjects<UserProfile>) {
```

Comment:
why is this being exported ?

Reply:
It's being used in `app/mm-redux/selectors/entities/groups.ts` line 72.

Reply:
Use getcurrentuserlocale selector.

Reply:
Done.